moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -134,7 +134,7 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
               data={data}
               classes={classes}
             />
-            <Grid container spacing={0} xs={12}>
+            <Grid container spacing={0}>
```

Comment:
I was getting errors in my console for these sizes on Grid containers, so I removed them.